packages/amplify_api/lib/graphql_helpers.dart

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_api_plugin_interface/amplify_api_plugin_interface.dart';
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+
+class ModelQueries extends ModelQueriesInterface {
+  @override
+  static GraphQLRequest<T> get<T extends Model>(
```

Comment:
Although, now I'm wondering if `ModelQueries` even needs an interface. Do you think we could put all the static methods on `ModelQueries` and remove the interface class?

Reply:
Thanks for the insight, I didn't realize dart operated like this. I want to keep the interface files because they feel like better code design, but to enable that I've converted this class to `ModelQueriesFactory` and added a `ModelQueries` class that provides static calls to the factory. Thanks for the feedback!